common/app/templates/inlineJS/blocking/shouldEnhance.scala.js

```diff
@@ -15,6 +15,17 @@
         }
     };
 
+    var personDemandsFeatures = function () {
```

Comment:
now we've settled on nomenclature, maybe this should be `personDemandsEnhanced`?